schemas/witnet/witnet.proto

```diff
@@ -280,6 +280,7 @@ message ConsensusConstants {
     uint32 reputation_issuance = 7;
     uint32 reputation_issuance_stop = 8;
     double reputation_penalization_factor = 9;
+    uint32 consensus_c = 10;
```

Comment:
Despite its name, this constant is not part of the consensus parameters: this means that two peers with a different `consensus_c` should be able to peer without any problems. Therefore, `consensus_c` should be stored in a different data structure, and should be loaded from config on every node startup.